Cargo.toml

```diff
@@ -24,10 +17,14 @@ travis-ci = { repository = "gltf-rs/gltf" }
 members = ["gltf-derive", "gltf-json", "gltf-importer", "gltf-utils"]
 
 [dependencies]
-approx = "0.1.1"
-cgmath = "0.15"
-gltf-json = { path = "gltf-json", version = "0.9.2" }
-lazy_static = "0.2"
+approx = "^0.1.1"
+cgmath = "^0.15.0"
+lazy_static = "^0.2.0"
+mint = { optional = true, version = "^0.4" }
```

Comment:
Why the carets? They're implicit (http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-cratesio)